### DISCUSSION TITLE
Does this library support cultures. Like en-uk and en-us?

### DISCUSSION THREAD

**Discussion by juanjoseluisgarcia** [sveltekit-i18n/lib#144] [Q&A] [answered]
I found out that culture is a very sensitive issue and different cultures like to see things in their own 'flavour' of the language and syntax. Is it possible to have different loaders based on this?

**Comment by Nycolaide** [comment]
Hello , i use sveltekit-i18n in my project and yes sveltekit-i18n support cultures.
`
/** @type {import('sveltekit-i18n').Config} */
const config = {
	fallbackLocale: defaultLocale,
	translations: {
		'en-US': { languages },
		'fr-FR': { languages }
	},
	loaders: [...translateEn, ...translateFr]
};
`

And my loaders :) 
<img width="209" alt="Screenshot 2023-09-06 at 11 13 14" src="https://github.com/sveltekit-i18n/lib/assets/62988176/1b20aeea-2f52-4028-8be9-09fb91a5c8a7">

Sorry for my English , i'm french :/

**Comment by juanjoseluisgarcia** [comment]
Apologies for the question. Would you mind to give me a schema on how the translateEn functions looks like? 
Thanks

**Comment by Nycolaide** [comment, marked as answer]
Oh yes! 


index.ts
```
const locale = 'en-us';
const translations = [
	{
		locale: locale,
		key: 'common',
		loader: async () => (await import('./common.json')).default
	},
	{
		locale: locale,
		key: 'global',
		loader: async () => await import('./global.json')
	},
	{
		locale: locale,
		key: 'auth',
		loader: async () => (await import('./auth.json')).default
	},
	{
		locale: locale,
		key: 'legal',
		loader: async () => (await import('./legal.json')).default
	}
];

export default translations;
```

I call translate in my +page.svelte =>  $t('auth.password')

[a comment by an automated account is not shown]
